packages/integration-tests/package.json

```diff
@@ -16,6 +16,7 @@
         "@cypress/skip-test": "^2.6.0",
         "@cypress/webpack-preprocessor": "^5.5.0",
         "cypress-image-snapshot": "^4.0.0",
+        "chrome-remote-interface": "^0.31.0",
```

Comment:
this allows us to use chrome dev tools from tests. for example to force state of dom element to :hover